contracts/bonding/BondingManager.sol

```diff
@@ -29,9 +29,7 @@ contract BondingManager is IBondingManager, Manager {
 
     // Represents a transcoder's current state
     struct Transcoder {
-        address transcoderAddress;                           // The address of this transcoder
```

Comment:
Yeah having the address in the struct in addition to using as the key in the mapping didn't seem necessary and removing it also saves some gas since there is one less store op